### DISCUSSION TITLE
What about color for text-decoration?

### DISCUSSION THREAD

**Discussion by frederikhors** [tailwindlabs/tailwindcss#2166] [Ideas]
What about color for text-decoration?

**Comment by simonswiss** [comment]
Hey @frederikhors - not sure I understand the question?

You can achieve "colored text-decoration" by composing a text color class and a text decoration class together: 

```html
<p class="text-red-600 line-through">
    I am red and crossed out
</p>
```

![image](https://user-images.githubusercontent.com/485747/91113571-45acf800-e6c9-11ea-8e92-acda1782e60b.png)

Is this what you're trying to do?

**Reply by DanielRiveraHQ** [reply]
Hey @simonswiss,

I believe that is not possible to have a different color from the text for underline or line-through
